### ISSUE TITLE
train报错

### ISSUE THREAD

**Issue by lhz-1** [PaddlePaddle/PaddleClas#3014]
023/10/23 19:57:42] ppcls WARNING: The training strategy provided by PaddleClas is based on 4 gpus. But the number of gpu is 1 in current training. Please modify the stategy (learning rate, batch size and so on) if use this config to train.
LAUNCH INFO 2023-10-23 19:57:59,699 Pod failed
LAUNCH ERROR 2023-10-23 19:57:59,700 Container failed !!!
AUNCH INFO 2023-10-23 19:57:59,714 ------------------------- ERROR LOG DETAIL -------------------------
LAUNCH INFO 2023-10-23 19:57:59,715 Exit code 3221226505

报错没头绪

**Comment by lhz-1** [comment]
[https://blog.csdn.net/Jacky__Lv/article/details/128048726?utm_medium=distribute.pc_relevant.none-task-blog-2~default~baidujs_utm_term~default-1-128048726-blog-128103505.235^v38^pc_relevant_sort_base1&spm=1001.2101.3001.4242.2&utm_relevant_index=4](url)
按照这个方法解决了
